core/scripts/validate_domain.py

```diff
@@ -9,48 +9,59 @@
 
 RESPONSES_KEY = "responses"
 BUTTONS_KEY = "buttons"
+TEXT_KEY = "text"
+TITLE_KEY = "title"
 PAYLOAD_KEY = "payload"
 
 
-def get_buttons(response: List[dict]) -> List[str]:
+def get_text(response: List[dict]) -> Set[str]:
     if len(response) != 1:
         raise Exception("This tool does not support checking response variants (yet).")
```

Comment:
Shouldn't that only be a warning?

Reply:
A warning will fail silently in the CI, if we want to use variants, we should update the tool accordingly and be notified as soon as possible.